src/aeromancy/runner.py

```diff
@@ -181,13 +197,24 @@ def store_environment_variables(
 def run_docker(
     env_filename: str,
     docker_tag: str,
-    docker_subcommand: str,
-    extra_docker_run_args: str,
+    docker_subcommand_pieces: list[str],
+    extra_docker_run_args: list[str],
 ) -> None:
     """Run `docker run` with specified arguments."""
+    local_cache_path = Path("~/Cache").expanduser()
     interactive(
-        f"docker run --env-file {env_filename} -v ~/Cache:/root/Cache "
-        f"{extra_docker_run_args} -it {docker_tag} {docker_subcommand}",
+        [
+            "docker",
+            "run",
+            "--env-file",
+            env_filename,
+            "-v",
+            f"{local_cache_path!s}:/root/Cache",
```

Comment:
rm "!s" (or comment why it has to be there :smiley:  )

Reply:
Done.